yaksh/templates/yaksh/grade_user.html

```diff
@@ -1,46 +1,102 @@
+     <link rel="stylesheet" href="{{ URL_ROOT }}/static/yaksh/css/gradeuser.css" type="text/css" /> 
 {% endblock %}
 
 {% block script %}
 <script src= "{{ URL_ROOT }}/static/yaksh/js/edit_question.js"></script>
 {% endblock %}
+
 {% block manage %}
 
-<p> 
-Name: {{ data.user.get_full_name.title }}
+
+{% if course_details %}
+<table id = "course-details">
+<tr>
+<th>Courses</th> 
+<th> Quizzes </th>
+</tr>
+
+{% for course in course_details %}
+<tr>
+<td><ul>{{course.name}} </td>
+
+{% if course.get_quizzes %}
+<td>
+{% for quiz in course.get_quizzes %}
+ <li><a href = "{{URL_ROOT}}/exam/manage/gradeuser/{{quiz.id}}">{{quiz.description}}</a></li>
+
+{% endfor %}
+</td>
+{% else %}
+<td> No quiz</td>
+{% endif %}
+</ul></tr>
+{% endfor %}
+</table>
+
+{% endif %}
+<div class="row">
+
+{%if users %}
+<div id = "student" class=span2>
+{% for user in users %}
+<p><a href = "{{URL_ROOT}}/exam/manage/gradeuser/{{quiz_id}}/{{user.id}}">{{user.get_full_name}} </a></p>
+{% endfor %}
+</div>
+{% endif %}
+
+
+
+
+{% if data %}
+<div class=span12 id = "paper">
+<p> <h3> <center> Showing paper for {{data.user.get_full_name.title}} </center></h3>
+<p><b>Name:</b>{{ data.user.get_full_name.title }}
 {% if data.profile %}
-(roll number: {{ data.profile.roll_number }}) <br/>
-Position: {{ data.profile.position }} <br/>
-Department: {{ data.profile.department }} <br/>
-Institute: {{ data.profile.institute }} <br/>
+
+<p><b> Roll number:</b> {{ data.profile.roll_number }}
+<p><b>Position: </b> {{ data.profile.position }}
+<p><b>Department: </b>{{ data.profile.department }} 
+<p><b>Institute: </b>{{ data.profile.institute }}
 {% endif %}
-</p>
 
 {% if data.papers %}
 
 {% for paper in data.papers %}
-
-{% if forloop.counter == 2 and data.questionpaperid %}
 <hr>
-<u><h2> Previous attempts </h2></u>
-{% endif %}
 {{ paper.total_marks }}
 
-<h2> Quiz: {{ paper.question_paper.quiz.description }} </h2>
+<h3> Quiz: {{ paper.question_paper.quiz.description }} </h3>
 
 <p>
-Attempt Number: {{ paper.attempt_number }}<br/>
-Questions correctly answered: {{ paper.get_answered_str }} <br/>
+Attempt Number: <b>{{paper.attempt_number}} </b>
+<select id = "attempt" onchange="window.location.href=this.value">
+<option selected="">Select attempt number</option>
+{%for attempt in attempts %}
+<option value = "{{URL_ROOT}}/exam/manage/gradeuser/{{quiz_id}}/{{user_id}}/{{attempt.attempt_number}}/"> 
+{{attempt.attempt_number}}
+</option>
+{% endfor %}
+</select>
+
+<br/>Questions correctly answered: {{ paper.get_answered_str }} <br/>
 Total attempts at questions: {{ paper.answers.count }} <br/>
-Marks obtained: {{ paper.get_total_marks }} <br/>
+Marks obtained: {{ paper.marks_obtained }} <br/>
 Start time: {{ paper.start_time }} <br/>
+{%if paper.percent%}
+Percentage obtained: {{paper.percent}}% <br/>
+{% endif %}
+{% if paper.passed == 0 %}
+Status : <b><font color = "red"> Failed </font></b><br/>
```

Comment:
Please use a suitable class without this non-standard element.  At least use a style attribute!